src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/ampSrcBuilder.test.js

```diff
@@ -0,0 +1,14 @@
+import ampSrcBuilder from './ampSrcBuilder';
+
+const includeSupportingAmp =
+  '/include/newsspec/21841-green-diet/gahuza/app?responsive=true&newsapps=true&app-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-asdf/app_launcher_ws_640_7ania.png&app-clickable=true&amp-clickable=true&amp-image-height=360&amp-image-width=640&amp-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-asdf/app_launcher_ws_640_7ania.png';
+
+describe('Amp Src Builder', () => {
+  process.env.SIMORGH_INCLUDES_BASE_AMP_URL = 'https://news.files.bbci.co.uk';
```

Comment:
This should be cleaned up at the end of the test as it is a change to the global scope that could have side-effects for other tests in the test suite.

Does this have an existing value before the test runs? If not, best to delete it: `delete process.env.   process.env.SIMORGH_INCLUDES_BASE_AMP_URL;` after the test completes.